src/OpenLoco/src/Ui/Widgets/ScrollViewWidget.cpp

```diff
@@ -8,232 +8,239 @@
 
 namespace OpenLoco::Ui::Widgets
 {
+    // For horizontal scrollbars its 10 wide, for vertical its 10 tall
+    static constexpr auto kScrollbarSize = 10;
+    static constexpr auto kScrollbarMargin = 1;
+
+    static constexpr auto kScrollButtonSize = Ui::Size(10, 10);
+    static constexpr auto kArrowOffset = 2;
+
     static void drawHScroll(Gfx::DrawingContext& drawingCtx, const Widget& widget, const WidgetState& widgetState, const ScrollArea& scrollArea)
     {
         auto* window = widgetState.window;
+        const auto position = window->position() + widget.position();
+        const auto size = widget.size();
+        const auto colour = widgetState.colour;
 
-        uint16_t left = window->x + widget.left + 1;
-        uint16_t top = window->y + widget.top + 1;
-        uint16_t right = window->x + widget.right - 1;
-        uint16_t bottom = window->y + widget.bottom - 1;
+        // Calculate adjusted dimensions
+        auto scrollPos = Ui::Point{ position.x + kScrollbarMargin, position.y + size.height - kScrollbarSize - kScrollbarMargin };
+        auto scrollSize = Ui::Size{ size.width - kScrollbarMargin, kScrollbarSize };
 
-        top = bottom - 10;
         if (scrollArea.hasFlags(Ui::ScrollFlags::vscrollbarVisible))
         {
-            right -= 11;
+            scrollSize.width -= kScrollbarSize + kScrollbarMargin;
         }
 
         Gfx::RectInsetFlags f;
         auto tr = Gfx::TextRenderer(drawingCtx);
 
-        // pusha
+        // Scroll track
+        drawingCtx.fillRect({ scrollPos.x, scrollPos.y }, scrollSize, Colours::getShade(colour.c(), 7), Gfx::RectFlags::none);
+        drawingCtx.fillRect({ scrollPos.x, scrollPos.y }, scrollSize, Colours::getShade(colour.c(), 3), Gfx::RectFlags::crossHatching);
```

Comment:
Could this not be written as:
```suggestion
        drawingCtx.fillRect(scrollPos, scrollSize, Colours::getShade(colour.c(), 7), Gfx::RectFlags::none);
        drawingCtx.fillRect(scrollPos, scrollSize, Colours::getShade(colour.c(), 3), Gfx::RectFlags::crossHatching);
```